Dockerfile

```
@@ -1,10 +1,16 @@
-FROM nvidia/cuda:8.0-devel-ubuntu16.04
+FROM nvidia/cuda:9.0-devel-ubuntu16.04
```

I think you can build on top of our existing cuda image?

```suggestion
FROM rajaorg/compiler:nvcc9
```